source/apps/crawl_space/crawl_supervisor.py

```diff
@@ -1,98 +0,0 @@
-"""Crawl Supervisor is a free-standing script that monitors and logs output
```

Comment:
Yes, these files were the files which were previously running the crawls. Just took them out as a side thought.